Lib/sqlite3/test/userfunctions.py

```diff
@@ -198,13 +182,10 @@ def setUp(self):
         self.con.create_function("memoryerror", 0, func_memoryerror)
         self.con.create_function("overflowerror", 0, func_overflowerror)
 
-        self.con.create_function("isstring", 1, func_isstring)
-        self.con.create_function("isint", 1, func_isint)
-        self.con.create_function("isfloat", 1, func_isfloat)
-        self.con.create_function("isnone", 1, func_isnone)
-        self.con.create_function("isblob", 1, func_isblob)
-        self.con.create_function("islonglong", 1, func_islonglong)
-        self.con.create_function("spam", -1, func)
+        self.con.create_function("isblob", 1,
+                                 lambda x: isinstance(x, (bytes, memoryview)))
```

Comment:
Why check for memoryview? Blobs are represented as bytes object in Python.

Reply:
> Why check for memoryview? Blobs are represented as bytes object in Python.

It's a leftover from the old code. I'll remove the memoryview check.